front/src/tools/templates/component/index.js

```diff
@@ -0,0 +1,2 @@
+import __name__ from "./__name__";
+export default __name__;
```

Comment:
Some programmers leave a blank line at the end of each file. But this is absolutely not necessary.